docs/json_data/sysfs_attributes_drm.json

```diff
@@ -539,6 +539,138 @@
             "units": "none",
             "writeable": false,
             "alias": ""
+        },
+        "ACT_FREQ": {
+            "attribute": "freq0/act_freq",
```

Comment:
Were you able to figure out what the 0 in `freq0` maps to in the xe driver? I wonder if it can report more than one frequency domain in those different numbers (vs how the i915 driver reports rps freq and base freq).

Reply:
Seems hardcoded 

https://github.com/torvalds/linux/blob/b5329d5a35582abbef57562f9fb6cb26a643f252/drivers/gpu/drm/xe/xe_gt_freq.c#L41

and

https://github.com/torvalds/linux/blob/b5329d5a35582abbef57562f9fb6cb26a643f252/drivers/gpu/drm/xe/xe_gt_freq.c#L251